include/hc.hpp

```diff
@@ -1470,7 +1471,8 @@ inline completion_future accelerator_view::create_blocking_marker(completion_fut
 
     // If necessary create an explicit dependency on previous command
     // This is necessary for example if copy command is followed by marker - we need the marker to wait for the copy to complete.
-    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr);
+    hc::memory_scope releaseScope = hc::no_scope;
+    std::shared_ptr<Kalmar::KalmarAsyncOp> depOp = pQueue->detectStreamDeps(hcCommandMarker, nullptr, &releaseScope);
```

Comment:
@bensander this is a place where we can consider make the 3rd parameter of `detectStreamDeps` be optional.